assessment.css

```diff
@@ -0,0 +1,42 @@
+body {
+    background-color: lavender;
+   
+    width: 100vh;
+   
+}
+
+h1 {
+    width: 70%;
+    height: 150px
+}
+
+.top {
+    display: grid;
```

Comment:
`display: flex` would have been easier here. I would prevent you from having to specify the columns that the header and image must span. You can talk to @lakerfan1994 (Chuck) to see how he approached this and what to do for our website to be more responsive.